tests/test_linearity.py

```diff
@@ -0,0 +1,93 @@
+# This file is part of ci_cpp.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import unittest
+import lsst.utils.tests
+
+
+class PtcTestCases(lsst.utils.tests.TestCase):
```

Comment:
I have the same question as above about "ill-defined", but, more generally, what are the plans for all these tests (including the ones in the other files) that say "pass"? Is there a time frame to implement these? Also, in DMTN-101 many of the tests specify that the data should be taken in a specific way: for example, for the non-linearity tests. I guess this does not have to necessarily be enforced for these tests, correct?

Reply:
I would like to have useful tests to know that the linearity is constructed correctly.  However, those tests have not been defined in a generic way.  If this package existed before now, we likely would have been adding those tests as the `cp_pipe` construction code was added.  I'm adding ticket DM-26396 that can be done in the future to add/update tests.  I am also adding reference to that ticket in all of the test_ scripts, so there is a note pointing to that future work.